### PR TITLE
ref(javascript): Streamline cache instrumentation page

### PR DESCRIPTION
**Pull request by Lms24**
- Remove SDK setup section as its already covered extensively in highler level guides
- Only show page on server-side SDKs
- Some wording adjustments

### CODE REVIEW

**Review by andreiborza — APPROVED** [review]

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
-If you're using anything other than Sentry's redis integration, you'll need to manually instrument the [Cache Module](<(https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/)>) by following the steps below.
+If you're using anything other than Sentry's Redis integration, you'll need to manually instrument the [Cache Module](<(https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/)>) by following the steps below.
 
 You'll need to create two spans - one indicating that something is being put into the cache, and a second one indicating that something is being fetched from the cache.
 
-Make sure that there's a transaction running when you create the spans. If you're using a web framework those transactions will be created for you automatically. See <PlatformLink to="/performance/">Performance Monitoring</PlatformLink> for more information.
-
-For detailed information about which data can be set, see the [Cache Module Developer Specification](https://develop.sentry.dev/sdk/performance/modules/caches/).
-
+<PlatformSection supported={["javascript.node", "javascript.deno", "javascript.bun"]}>
```
Comment (mydea) [inline docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:34]:
```suggestion
<PlatformCategorySection supported={["server"]}>
```

I think this is better, or do we not want to show this for fastify etc?

Comment (mydea) [reply docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:34]: hmm or I guess, this is fine, should only be shown for framework-less SDK?

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx [hunk 2]
```diff
@@ -1,34 +1,47 @@
...
 
+</PlatformSection>
 
-### Initialize Sentry
-
-<PlatformContent includePath="getting-started-config" />
+For detailed information about which data can be set, see the [Cache Module Developer Specification](https://develop.sentry.dev/sdk/performance/modules/caches/).
 
-### Add Span When Putting Data Into the Cache
+### 1. Add Span When Putting Data Into the Cache
```
Comment (mydea) [inline docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:42]: If this is 1., we also should add numbers to the headings below!

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx (hunk 1 above)
Comment (Lms24) [reply docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:34]: yeah that was my thinking. Doesn't make sense in guides where we already have auto-instrumented request spans

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx (hunk 2 above)
Comment (Lms24) [reply docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:42]: whoops good catch, thanks! I was thinking numbering helps here, given users _should_ do both and numbering helps conveying this.

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx [hunk 3]
```diff
@@ -61,7 +74,7 @@ Sentry.startSpan(
 );
 ```
 
-### Add Span When Retrieving Data From the Cache
+### 2.Add Span When Retrieving Data From the Cache
```
Comment (lizokm) [inline docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:77]:
```suggestion
### Step 2: Add Span When Retrieving Data From the Cache
```
Same comment as above :)

**Review by lizokm — APPROVED** [review]
Looks good to me, just added a few small suggestions. Thanks for updating!!

File: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx [hunk 4]
```diff
@@ -1,34 +1,47 @@
...
-
+<PlatformSection supported={["javascript.node", "javascript.deno", "javascript.bun"]}>
 
+Make sure that there's an active span before you create your cache spans. If you're using a web framework like Express, such a span will be created for you automatically. See <PlatformLink to="/performance/">Performance Monitoring</PlatformLink> for more information.
 
+</PlatformSection>
 
-### Initialize Sentry
-
-<PlatformContent includePath="getting-started-config" />
```
Comment (DominikB2014) [inline docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/cache-module.mdx:27]: We should be consistent here and ensure the python section also gets removed,
I've created a PR to do so!
https://github.com/getsentry/sentry-docs/pull/10135

**Review by DominikB2014 — COMMENTED** [review]
Do you mind renaming `cache-module.mdx` to `caches-module.mdx`.
The module name is actually plural, this was a typo on my part.

**Comment by Lms24** [comment]
Thx for the reviews, everyone!

**Pull request merged**